example/secondary-axes.md

```diff
@@ -0,0 +1,39 @@
+# Secondary Axes
+
+When x2 values exist in your data or a `.x2` accessor is supplied, these values will be plotted on the secondary x-axis and when y2 values exist in your data or a `.y2` accessor is supplied, these values will be plotted on the secondary y-axis.
+
+```js
+var data = [
+  {id: "alpha", alphaX: 4, alphaY:  7},
+  {id: "alpha", alphaX: 5, alphaY: 25},
+  {id: "alpha", alphaX: 6, alphaY: 13},
+  {id: "beta",  betaX: 28, betaY: 700},
+  {id: "beta",  betaX: 33, betaY:  850},
+  {id: "beta",  betaX: 56, betaY: 900}
+];
+```
+
+We can create a simple Line Plot with a secondary x-axis and secondary y-axis:
+
+```js
+new d3plus.LinePlot()
+  .data(data)
+  .groupBy("id")
+  .x(d => d["alphaX"])
+  .x2(d => d["betaX"])
+  .xConfig({
+    title: "alpha"
+  })
+  .x2Config({
+  title: "beta"
+  })
+  .yConfig({
+  title: "alpha"
+  })
+  .y2Config({
+  title: "beta"
+  })
+  .y(d => d["alphaY"])
+  .y2(d => d["betaY"])
+  .render();
```

Comment:
Let's simplify this example:

- [ ] change the accessors to just be strings (ie. `d => d["alphaX"]` becomes just `"alphaX"`
- [ ] remove the axis configs for changing the titles